manifests/monitor/monitor-configMap.yaml

```diff
@@ -0,0 +1,181 @@
+# Source: tidb-cluster/templates/monitor-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: external-config
+  labels:
+    app.kubernetes.io/name: tidb-cluster
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/instance: test
+    app.kubernetes.io/component: monitor
+    helm.sh/chart: tidb-cluster-dev
```

Comment:
What's the difference between this configuration and the configuration auto-generated by tidbmonitor? I don't think we should make mounting external configuration as the default choice for tidbmonitor.